massgov/mayflower
# DP-16028 Fix video transcription link display

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Adjust the video transcription link width to line up one line without overwrapping to video.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-16028)

## Steps to Test


1. Check the transcript link is displayed as one line and no overwrap to video:
- /patterns/01-atoms-09-media-video/01-atoms-09-media-video.html
- /patterns/05-pages-marketing-campaign/05-pages-marketing-campaign.html
- /patterns/05-pages-marketing-campaign-as-solid/05-pages-marketing-campaign-as-solid.html
- /patterns/05-pages-press-release-with-video/05-pages-press-release-with-video.html
- /patterns/05-pages-service/05-pages-service.html

## Screenshots
Before:
<img width="442" alt="Pattern_Lab_-_pages-service" src="https://user-images.githubusercontent.com/9633303/66347771-16f82880-e923-11e9-953b-fb1aacf6fbb2.png">

After:
<img width="631" alt="Pattern_Lab_-_atoms-video" src="https://user-images.githubusercontent.com/9633303/66347775-195a8280-e923-11e9-834c-0ee1f5e372b4.png">


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
